README.md

```diff
@@ -1,2 +1,9 @@
 # mailroom-madness
 CodeFellows 401, Day 4, Mailroom Madness Assignment
+
+
+Got help testing name input function from:
+http://stackoverflow.com/questions/6271947/how-can-i-simulate-input-to-stdin-for-pyunit
+
+Got help testing stdout from here:
+http://stackoverflow.com/questions/4219717/how-to-assert-output-with-nosetest-unittest-in-python
```

Comment:
I'd like to see some description of the scope and goals of the project.  What does it do?